spring-kafka-docs/src/main/antora/modules/ROOT/pages/testing.adoc

```diff
@@ -56,8 +56,7 @@ If this is not possible for some reason, note that the `consumeFromEmbeddedTopic
 Since it does not have access to the consumer properties, you must use the overloaded method that takes a `seekToEnd` boolean parameter to seek to the end instead of the beginning.
 ====
 
-NOTE: The `EmbeddedKafkaRule` JUnit 4 rule has been removed in version 4.0.
-For JUnit 4, you should use the `EmbeddedKafkaKraftBroker` directly or migrate to JUnit 5 with the `@EmbeddedKafka` annotation.
+NOTE: `Spring Kafka` no longer supports JUnit4. Please consider migrating from JUnit4 to JUnit5.
```

Comment:
I would not refer to JUnit Jupiter as "JUnit 5", especially since we (the JUnit team) will be releasing JUnit Jupiter 6 as part of "JUnit 6" later this year.

In other words, I think it is better to refer to JUnit Jupiter as "JUnit Jupiter" to avoid any confusion and unnecessary updates to documentation in the future.

I would also suggest that "JUnit Jupiter" be used consistently across the documentation.